src/components/Sidebar.js

```diff
@@ -15,16 +24,38 @@ class Sidebar extends Component {
     }
   }
 
+  componentWillReceiveProps (nextProps) {
+    this.setState({
+      items: nextProps.stacks.items
+    })
+  }
+
+  filterStacks (e) {
+    const query = e.currentTarget.value
+    const $items = this.props.stacks.items.filter((item) => {
```

Comment:
`$items` are used only when the array contains react elements, which it doesn't in this case. Remove the `$` please